ci/playbooks/multinode-customizations.yml

```diff
@@ -171,9 +171,11 @@
         state: present
 
     - name: Ensure we know compute host keys
-      ansible.builtin.shell:
-        cmd: |
-          ssh-keyscan {{ hostvars[item].ansible_host }} >> ~/.ssh/known_hosts
```

Comment:
I think we "just" need to set the `-t <type>` for ssh-keyscan to NOT try to get RSA keys - or any non-FIPS compatible type?

Reply:
Already tried, without success. Might be some other detail that is hard to debug since sshd fails without too much details (not even with debug level set).

Reply:
sshd itself? "nice". Is this something we may debug later using the reproducer? is anything needed on that field actually, other than this fix/workaround? Since we have console access...